pyomo/core/expr/numeric_expr.py

```diff
@@ -1303,8 +1303,10 @@ def _to_string(self, values, verbose, smap, compute_values):
             tmp = [self.constant.to_string(compute_values=False)]
         if verbose:
             for c,v in zip(self.linear_coefs, self.linear_vars):
-                if smap:                        # TODO: coverage
+                if smap and not v.fixed:
                     v_ = smap.getSymbol(v)
+                elif v.fixed:
+                    v_ = str(v.value)
```

Comment:
I think we need to re-think this.  This changes the string representation of *all* Pyomo LinearExpressions to replace Vars with their values when they are fixed.  We probably don't want to do this.  Unfortunately, I am not sure how to (easily) fix this without changing the `_to_string` API.

Reply:
I put a fix in the GAMS writer visitor instead.